routes/consumption_history.py

```diff
@@ -39,7 +39,7 @@ def get_readings(meter_id):
     for key in get_sorted_keys(meter_id):
         data = json.loads(redis_client.get(key))
         if data.get('type') == 'reading':
-            timestamp = key[len(meter_id):]
```

Comment:
Please merge master into your branch and check which actual changes you are proposing.